definitions/reports/advisor_on_prem_remediations.rb

```diff
@@ -0,0 +1,24 @@
+module Reports
+  class AdvisorOnPremRemediations < ForemanMaintain::Report
+    metadata do
+      description 'Report number of remediations based on advisor rules from advisor on premise'
+      confine do
+        feature(:installer)&.answers&.dig('foreman::plugin::rh_cloud', 'enable_iop_advisor_engine')
```

Comment:
By having this here, the key-value pair is omitted from the resulting report completely if iop advisor is disabled. Personally I'd prefer to have the key there with nil value if it is not enabled at all. @ShimShtein opinions?

Reply:
Let's have two fields: 
boolean `advisor_on_prem_remediations_enabled` and `advisor_on_prem_remediations_count` that will constantly be present in the report. 
I don't think we should make fields optional. It will make the summary queries much more complex.

Reply:
Applied.